test/helpers/ProposalTest.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {
+  ICompoundTimelock
+} from "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {IERC20} from "@openzeppelin/contracts/interfaces/IERC20.sol";
+
+import {Propose} from "script/Propose.s.sol";
+import {IGovernorAlpha} from "src/interfaces/IGovernorAlpha.sol";
+import {PooltogetherGovernorTest} from "test/helpers/PooltogetherGovernorTest.sol";
+
+abstract contract AlphaProposalTest is PooltogetherGovernorTest {
```

Comment:
Since the contract is called `AlphaProposalTest`, should we name the file the same?

Reply:
Good question, after moving a little past this I don't think we need the Alpha here. `ProposalTest` will do and I will update.

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/11/commits/c479e14f99c451b7b828025671756864e01e948e